changelog.dd

```diff
@@ -175,6 +176,10 @@ $(BUGSTITLE Compiler Changes,
         }
         ---
     )
+
+    $(LI $(LNAME2 dash_safe, Add -transition=safe switch.)
+	$(P Enables enhanced @safe checking, which will break some existing code.)
```

Comment:
What they will specifically be is unknown at the moment, because it depends on which of the @safe improvements will break significant amounts of code. The only thing that is clear is that we'll need this switch.